articles/aks/ssh.md

```diff
@@ -92,16 +95,10 @@ aksnpwin000000                      Ready    agent   87s     v1.19.9   10.240.0.
 
 In the above example, *10.240.0.67* is the internal IP address of the Windows Server node.
 
-Return to the terminal started by `kubectl debug` and update the permission of the private SSH key you copied to the pod.
-
-```azurecli-interactive
-chmod 0400 id_rsa
-```
-
 Create an SSH connection to the Windows Server node using the internal IP address. The default username for AKS nodes is *azureuser*. Accept the prompt to continue with the connection. You are then provided with the bash prompt of your Windows Server node:
 
 ```output
-$ ssh -i id_rsa azureuser@10.240.0.67
+$ ssh -o 'ProxyCommand ssh -p 2022 -W %h:%p azureuser@127.0.0.1' azureuser@10.240.0.67
```

Comment:
does it not work with a password?

Reply:
Yeah, could you please also provide a sample to use user:pass as well? @jsturtevant  I know we did not have this case in the previous doc but many cxs prefer password to log onto the windows node, including me :)
Thanks.